### ISSUE TITLE
Edge addons support

### ISSUE THREAD

**Issue by ghost** [ivebencrazy/favioli#64]
It would be great if this extension was available on Edge add-ons as well, it's for the Microsoft edge browser
https://microsoftedge.microsoft.com/addons

**Comment by bpevs** [comment]
oh strange, does edge not allow just install from the chrome store?  Or do you just mean to make the chromium ext available on that store for convenience? I'm not using windows much atm, so I'm unfamiliar with the ecosystem.

**Comment by ghost** [comment]
Yes, about making the chromium extension available on that store due to convenience
